Source/WebKit/NetworkProcess/NetworkSession.cpp

```diff
@@ -223,19 +223,22 @@ void NetworkSession::destroyPrivateClickMeasurementStore(CompletionHandler<void(
     privateClickMeasurement().destroyStoreForTesting(WTFMove(completionHandler));
 }
 
-void NetworkSession::setTrackingPreventionEnabled(bool enable)
+void NetworkSession::setTrackingPreventionEnabled(bool enabled)
 {
     ASSERT(!m_isInvalidated);
+    bool currentEnabled = !!m_resourceLoadStatistics;
```

Comment:
`currentEnabled` sounds weird in English and boolean variables need a prefix as per WebKit coding style. I suggest `isCurrentlyEnabled`